stetho-repl-js/README.md

```diff
@@ -0,0 +1,223 @@
+# Stetho's Javascript Console
+
+This [Stetho](https://facebook.github.io/stetho) plugin adds a javascript console by embedding Mozilla's [Rhino](https://github.com/mozilla/rhino).
+
+## Set-up
+
+### Download
+Download [the latest JARs](https://github.com/facebook/stetho/releases/latest) or grab via Gradle:
+```groovy
+compile 'com.facebook.stetho:stetho-repl-js:1.1.1'
+```
+or Maven:
+```xml
+<dependency>
+  <groupId>com.facebook.stetho</groupId>
+  <artifactId>stetho-repl-js</artifactId>
+  <version>1.1.1</version>
+</dependency>
+```
+
+Make sure that you depend on the main `stetho` dependency too.
+
+### Putting it together
+The javascript integration is similar to standard Stetho integration.
+The main difference is with the WebKitInspector used.
+There is a simple initialization step which occurs in your `Application` class:
+
+```java
+public class MyApplication extends Application {
+  public void onCreate() {
+    super.onCreate();
+    JavaScriptRuntimeBuilder jsRuntimeBuilder = new JavaScriptRuntimeBuilder(this);
+    Stetho.initialize(
+        Stetho.newInitializerBuilder(this)
+            .enableDumpapp(Stetho.defaultDumperPluginsProvider(this))
+            .enableWebKitInspector(jsRuntimeBuilder.javaScriptInspectorModulesProvider())
```

Comment:
Sigh, I really need to clean up this crappy extensibility API :)

I'm curious for your thoughts on something like this:

```java
.enableWebKitInspector(new CustomInspectorModulesProvider() {
  @Override
  public List<DevtoolsDomain> overrideModules() {
    return new Collections.singletonList(new Runtime(new JavaScriptRepl(context));
  }
})
```

I know it's more verbose, but it's more flexible and I think a bit more intuitive than the hacky stuff we've been recommending up until now.